index.html

```diff
@@ -353,11 +367,17 @@ <h3>
 Update
           </h3>
           <p>
-To update the DID document, the <code>did.json</code> has to be updated.
+To update the DID document, the associated file or files have to be updated.
```

Comment:
single file currently, no support for multiple representations, JSON only... for now at least.

Reply:
I disagree.

Index files are configured server-side, and we should leave that capability in place.

Reply:
Index files have always been with us. <a href="https://en.wikipedia.org/wiki/Web_server_directory_index">Wikipedia</a> has a historic overview.

The use case here is to comply with the core spec on various representations without breaking other long-standing best practices on filename extension and content type header agreement.